src/theme/components/card.ts

```diff
@@ -396,6 +396,14 @@ export const Card = defineMultiStyleConfig({
         borderRadius: 'none',
         borderBottom: '1px solid rgb(229, 229, 229)',
         backgroundColor: 'transparent',
+        py: { base: 6, lg: 8 },
+
+        '&:last-of-type': {
+          border: 'none',
+        },
+        '&:nth-last-of-type(2)': {
+          border: { base: 'none', lg: '1px solid rgb(229, 229, 229)' },
+        },
```

Comment:
I don't think this is what you actually wanted here
![imatge](https://github.com/user-attachments/assets/9ca5980c-1e5d-45ec-90d9-31abe5a87f78)